pyproject.toml

```diff
@@ -65,3 +65,41 @@ ignore_errors = true
 
 [tool.coverage.html]
 directory = "htmlcov"
+
+[tool.mypy]
+python_version = "3.9"
+namespace_packages = true
+files = [
+  "smartsim"
+]
+plugins = []
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+# Ignore packages that are not used or not typed
+module = [
+  "coloredlogs",
+  "smartredis",
+  "smartredis.error"
+]
+ignore_missing_imports = true
+ignore_errors = true
+
+[[tool.mypy.overrides]]
```

Comment:
Does it cause problems to have two tool.mypy.overrides sections?

Reply:
Nope, they allow us to apply different settings to different bits of target code. Here I'm just ignoring 3rd parties without stubs in one. The other is using a whitelist of the modules I want to include as required items to pass mypy.

In my next PR, I'll reverse the global ignore/local include pattern and the second overrides here will go away since it won't be necessary.